scripts/hudson/jenkins.sh

```diff
@@ -74,40 +74,56 @@ then
     rm -rf tmp
 fi
 
-GIT_BRANCH=master THREAD_COUNTS="1 24 240 1600" COMPARISON="com.arjuna.ats.jta.xa.performance.*StoreBenchmark.*" COMPARISON_COUNT=4 BM_LINE_PATTERN="(com.arjuna.ats.jta.xa.performance|i.n.p.p)" PATTERN2="Benchmark" narayana/scripts/hudson/jenkins.sh
-[ $? = 0 ] || fatal "Store benchmark failed"
-mv benchmark-output.txt benchmark-store-output.txt
-mv benchmark.png benchmark-store.png
+if [ -z $COMPARE_STORES ] || [ $COMPARE_STORES == "y" ];
```

Comment:
Did you mean `[ -z $COMPARE_STORES ] || COMPARE_STORES = "y"`

BTW there is a bash syntax for this already: ${COMPARE_STORES:-y}

Reply:
Ah wait, ignore that, you are doing a test (the ; threw me).

Reply:
The semi colon looks odd unless you put the `then` on the same line.

Reply:
I copied it from elsewhere in the file, would you like it changing throughout the file?

Reply:
I think there was one instance that had it on the next line (maybe one I previously added). Anyway I aligned them all

Reply:
in that file